### PR TITLE
fix: egress numbers in domains should be optional

### PR DESCRIPTION
**Pull request by efraa**

### CODE REVIEW

File: src/mods/domains/components/creation-editing/CreationEditingDomain.tsx [hunk 1]
```diff
@@ -25,7 +25,8 @@ export const CreationEditingDomain = () => {
       isEdit
         ? defaultValues
         : {
-            egressRule: '.*',
```
Comment (psanders) [inline src/mods/domains/components/creation-editing/CreationEditingDomain.tsx:28]: The `.*` is meant to allow outbound calls too all destination, and empty string would prevent users from making calls.

**Review by psanders — COMMENTED** [review]
I'm not sure if this was push to staging, but I'm still seeing the old behavior.

Screenshot 2023-01-09 at 9 18 41 PM

File: src/mods/domains/components/creation-editing/CreationEditingDomain.tsx (hunk 1 above)
Comment (efraa) [reply src/mods/domains/components/creation-editing/CreationEditingDomain.tsx:28]: You can send this rule, it's just not required.

Screen Shot 2023-01-11 at 9 37 10 AM


Screen Shot 2023-01-11 at 9 38 07 AM

**Review by psanders — APPROVED** [review]
LGTM

**Pull request merged**